### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by ivaan1992**
-Added new functions to the portfolio.
-Edited HTML file. 
-Edited css files. 
-Edited javascript file.
-Added images, buttons, and text.
-Fixed linters issues.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ const menu = document.querySelector('.menu');
 const ham = document.querySelector('.ham');
 const xIcon = document.querySelector('.xIcon');
 const menuIcon = document.querySelector('.menuIcon');
+const buttons = document.querySelectorAll('.button');
+const exitButtons = document.querySelectorAll('.x');
+const pupup = document.querySelector('#popup');
```
Comment (carlos-ssh) [inline script.js]: **Kindly work on these requested changes:**
1. The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
2. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie:

```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

1. - The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.
2. - Modal popup data is populated with the data from the projects array.
3. - Thanks to the previous 2 points, adding projects in the future will be easy. Students will only add it in one place - as another object in the projects array.
4. - The JavaScript code is implemented following the best practices.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @ivaan1992 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -164,23 +164,21 @@ <h3 class="work down">Work</h3>
...
-
-
-                
-        
+        </div>
         </section>
+
        
-        <div class="gray">
+       <div id="popup-project" class="popup-project"></div>
```
Comment (SajjadAhmad14) [inline index.html:171]: [ OPTIONAL BUT RECOMMENDED] Pop-up should be in the center of the page both for small and large viewports. Please see the screenshots below and fix the issues.
pop-up-desktop
pop-up-small

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi Ivan Linares Gaona, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**